README.md

```diff
@@ -124,7 +124,41 @@ server.route({
     }
 });
 ```
+### Custom `uri` template values
+    
+When using the `uri` option, there are several **default** template values that can be injected from the incoming request:
+
+* `{protocol}`
+* `{host}`
+* `{port}`
+* `{path}`
+    
+```javascript
+server.route({
+    method: 'GET',
+    path: '/',
+    handler: {
+        proxy: {
+            uri: '{protocol}:{port}//{host}/use/the/{path}'
```

Comment:
I'm not sure what you're trying to create an example here. I think this returns 

`http:8080//example/use/the/`